modules/460-log-shipper/crds/cluster-log-destination.yaml

```
@@ -552,3 +552,52 @@ spec:
                     anyOf:
                       - pattern: '^[a-zA-Z0-9_\-]+$'
                       - pattern: '^\{\{\ [a-zA-Z0-9\\\-][a-zA-Z0-9\[\]_\\\-\.]+\ \}\}$'
+                buffer:
+                  type: object
+                  oneOF:
+                    - properties:
+                        disk: {}
+                        type:
+                          enum:
+                            - Disk
+                      required:
+                        - disk
+                        - type
+                        - whenFull
+                    - properties:
+                        memory: {}
+                        type:
+                          enum:
+                            - Memory
+                      required:
+                        - memory
+                        - type
+                        - whenFull
+                  properities:
+                    type:
+                      type: string
+                      description: The type of buffer to use.
+                      enum: ["Disk", "Memory"]
+                    disk:
+                      type: object
+                      properities:
+                        maxSizeBytes:
```

```suggestion
                        maxSizeBytes:
```
This is not `Bytes` anymore. 
1. Just maxSize?
2. Examples are required and descriptions should be rewritten.